ext/GeoInterfaceTablesExt.jl

```diff
@@ -0,0 +1,129 @@
+module GeoInterfaceTablesExt
+
+using GeoInterface
+using GeoInterface.Wrappers
+using Tables
+
+# This module is meant to extend the Tables.jl interface to features and feature collections, such that they can be used with Tables.jl.
+# This enables the use of the Tables.jl ecosystem with GeoInterface wrapper geometries.
+
+# First, define the Tables interface
+
+Tables.istable(::Type{<: Wrappers.FeatureCollection}) = true
+Tables.isrowtable(::Type{<: Wrappers.FeatureCollection}) = true
+Tables.rowaccess(::Type{<: Wrappers.FeatureCollection}) = true
+Tables.rows(fc::Wrappers.FeatureCollection{P, C, E}) where {P <: Union{AbstractArray{<: Wrappers.Feature}, Tuple{Vararg{<: Wrappers.Feature}}}, C, E} = GeoInterface.getfeature(fc)
+Tables.rows(fc::Wrappers.FeatureCollection) = Iterators.map(Wrappers.Feature, GeoInterface.getfeature(fc))
+Tables.schema(fc::Wrappers.FeatureCollection) = property_schema(GeoInterface.getfeature(fc))
+
+# Define the row access interface for feature wrappers
+function Tables.getcolumn(row::Wrappers.Feature, i::Int)
+    if i == 1
+        return GeoInterface.geometry(row)
+    else
+        return GeoInterface.properties(row)[i-1]
+    end
+end
+Tables.getcolumn(row::Wrappers.Feature, nm::Symbol) = nm === :geometry ? GeoInterface.geometry(row) : Tables.getcolumn(GeoInterface.properties(row), nm)
+Tables.columnnames(row::Wrappers.Feature) = (:geometry, propertynames(GeoInterface.properties(row))...)
+
+# Copied from GeoJSON.jl
+# Credit to [Rafael Schouten](@rafaqz)
+# Adapted from JSONTables.jl jsontable method
+# We cannot simply use their method as we have concrete types and need the key/value pairs
+# of the properties field, rather than the main object
+# TODO: Is `missT` required?
+# TODO: The `getfield` is probably required once
+missT(::Type{Nothing}) = Missing
+missT(::Type{T}) where {T} = T
+
+function property_schema(features)
```

Comment:
Will this work for GeoJSON too? Would be good to delete the code there if it does